### PR TITLE
Fix guides

### PR DESCRIPTION
**Pull request by jmoreira-valory**
## Proposed changes

Related improvements to quickstart and on-chain protocol.

- Fix guides commands error.
- Homogenized quickstart section to match demo sections.
- Added more context to "register packages on-chain". 
- Fixed a few typos.

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/macros/doc_macros.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
```
Comment (dvilelaf) [inline docs/macros/doc_macros.py:6]: You're adding code to the docs folder, so ensure that our linters are processing this folder as well. They are defined in tox.ini. The ones you need are:
`isort, black, safety, bandit, gitleaks, black-check, isort-check,  flake8 , mypy , pylint , vulture , darglint`

Comment (jmoreira-valory) [reply docs/macros/doc_macros.py:6]: I was not sure where to place doc macros...  What is the suggested folder? scripts ?

Comment (dvilelaf) [reply docs/macros/doc_macros.py:6]: If they have to do with docs, I think it makes sense to have it where you placed it, but we need to update the linter commands to also look into the new directory.

Comment (jmoreira-valory) [reply docs/macros/doc_macros.py]: Moved to /scripts (simpler option). If @DavidMinarsch suggests other action, will fix it.

File: mkdocs.yml [hunk 2]
```diff
@@ -247,7 +247,8 @@ nav:
 
 
 plugins:
-  - macros
+  - macros:
```
Comment (DavidMinarsch) [inline mkdocs.yml:250]: What was the rational for adding this?

File: docs/guides/create_service_existing_agent.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+{{check_untagged_doc()}}
```
Comment (DavidMinarsch) [inline docs/guides/create_service_existing_agent.md:1]: What's this about? I don't get it

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]

File: docs/guides/create_service_existing_agent.md (hunk 3 above)
Comment (jmoreira-valory) [reply docs/guides/create_service_existing_agent.md:1]: The idea is to get a Warning on top of the page whenever the docs are compiled outside a tagged version. The 'macros' package allows to execute code inside the docs on compile-time.

**Review by dvilelaf — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**